hypertrace-trace-enricher/hypertrace-trace-enricher-impl/src/test/java/org/hypertrace/traceenricher/enrichment/enrichers/HttpAttributeEnricherTest.java

```diff
@@ -52,7 +54,7 @@ public void test_withAValidUrl_shouldEnrichHttpPathAndParams() {
 
     String locationParam = SpanAttributeUtils.getStringAttribute(e,
         Constants.getEnrichedSpanConstant(HTTP_REQUEST_QUERY_PARAM) + ".location");
-    assertEquals("", locationParam);
+    assertNull(locationParam);
```

Comment:
Assuming that downstream is handling null correctly.

Reply:
null here means the params wasn't set into the attribute map